tests/packagedcode/data/pypi/site-packages/codebase/lib/python3.9/site-packages/click-8.0.4.dist-info/RECORD

```diff
@@ -5,37 +5,4 @@ click-8.0.4.dist-info/RECORD,,
 click-8.0.4.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
 click-8.0.4.dist-info/top_level.txt,sha256=J1ZQogalYS4pphY_lPECoNMfw0HzTSrZglC4Yfwo4xA,6
 click/__init__.py,sha256=bOKrvMqmR9rN07vN_ycyrdF-EcTCl-EmuAjq-Fp4yPM,3243
-click/__pycache__/__init__.cpython-36.pyc,,

```

Comment:
note that these are added at installation time to RECORD... removing all these means your RECORD file is closer to that of RECORD in a .whl archive and not like the one from a site-packages installed wheel but I guess this is OK since you have a pip example of an as-installed wheel below